src/Tasks/Common/ConflictResolution/FrameworkListReader.cs

```diff
@@ -98,7 +98,7 @@ private static IEnumerable<ConflictItem> LoadConflictItems(string frameworkListP
                 }
 
                 ret.Add(new ConflictItem(assemblyName + ".dll",
-                                                packageId: null,
+                                                packageId: "TargetingPack",
```

Comment:
just question, when is this get used? Do we match "TargetingPack" and do something somewhere

Reply:
As part of eliminating the heuristic to find the package ID based on a path in NuGetUtils, I was going through and trying to make sure all items where we would want to find the package ID had package ID metadata set on them.